--- shortuuid/tests.py	2014-04-18 19:00:18.000000000 +0200
+++ shortuuid/tests.py	2018-10-10 19:31:17.460103444 +0200
@@ -1,7 +1,6 @@
 import os
 import sys
 import unittest
-import pep8
 
 from uuid import UUID, uuid4
 
@@ -94,16 +93,6 @@
         self.assertRaises(ValueError, su1.set_alphabet, "1")
         self.assertRaises(ValueError, su1.set_alphabet, "1111111")
 
-    def test_pep8(self):
-        pep8style = pep8.StyleGuide([['statistics', True],
-                                     ['show-sources', True],
-                                     ['repeat', True],
-                                     ['paths', [os.path.dirname(
-                                         os.path.abspath(__file__))]]],
-                                    parse_argv=False,
-                                    config_file=True)
-        report = pep8style.check_files()
-        assert report.total_errors == 0
 
 if __name__ == '__main__':
     unittest.main()
